plotly/plotly/plotly.py

```diff
@@ -1576,6 +1575,39 @@ def bad_create_animations(fig, kwargs, payload):
     return r_dict
 
 
+def get_uid_by_col_name(grid_url, col_name):
```

Comment:
The `grid_url` arg should be a little more flexible. Look at  https://github.com/plotly/plotly.py/blob/master/plotly/plotly/plotly.py#L318. I'm OK with this as is. But I could see it being frustrating to be able to reference files one way in one place and another way in another place.